lib/utils/display.js

```diff
@@ -357,10 +362,7 @@ class Display {
         const [res, rej, p] = args
         return input.start(() => p()
           .then(res)
-          .catch(rej)
-          // Any call to procLog.input.read will render a prompt to the user, so we always
-          // add a single newline of output to stdout to move the cursor to the next line
-          .finally(() => output.standard('')))
```

Comment:
We still need this for all other input prompts, including the initial password prompt.

Reply:
do we? for example, with `npm init`, when I hit `enter`, it jumps to a newline.